### PR TITLE
Documentation update only

### PR DESCRIPTION
**Pull request by erinn**
This is only a documentation update. While working with this module the lack of documentation drove me crazy. This isn't perfect but it is better.

### CODE REVIEW

**Comment by mcanevet** [comment]
@erinn can you rebase and squash your commits so that I can merge this?

File: README.md [hunk 1]
```diff
@@ -10,50 +10,333 @@ This module is provided by [Camptocamp](http://www.camptocamp.com/)
 This module requires Augeas.
 
 ## Simple usage
-
+```
     include postfix
```
Comment (raphink) [inline README.md]: If you use `````, there's no need to indent the code.

Comment (mcanevet) [inline README.md]: And you should use ```puppet to have proper syntax coloration

**Comment by mcanevet** [comment]
@erinn rebased and merged

**Pull request closed without merging**